Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEventListener.java

```diff
@@ -732,6 +730,22 @@ public void onChat(AsyncPlayerChatEvent event) {
             }
         }
         String message = event.getMessage();
+
+        PlayerPlotChatEvent plotChatEvent = this.eventDispatcher.callChat(plotPlayer, plot, message, plotRecipients, spies);
+
+        if (plotChatEvent.getEventResult() == Result.DENY) {
+            return;
+        }
+
+        boolean force = plotChatEvent.getEventResult() == Result.FORCE;
+
+        if (force || (plot.isDenied(plotPlayer.getUUID()) && !Permissions
```

Comment:
I think this should be `!force`, as the plot chat should happen if the result is `FORCE`